.github/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Version 0.9.0-dev
+# Version 0.10.0-dev
```

Comment:
You don't want to add the new `0.10.0-dev` section until _after_ the release has been made 🙂 So the V 0.9.0 should still be at the top.